scripts/publish-pypi.ts

```diff
@@ -0,0 +1,85 @@
+import * as argparse from 'argparse';
+import { checkoutReleaseBranch, question } from './release-util';
+const TMP_DIR = '/tmp/tfjs-pypi';
+import * as shell from 'shelljs';
+import { execSync } from 'child_process';
+import chalk from 'chalk';
+import * as fs from 'fs';
+
+const parser = new argparse.ArgumentParser();
+parser.addArgument('--git-protocol', {
+  action: 'storeTrue',
+  help: 'Use the git protocol rather than the http protocol when cloning repos.',
+});
+parser.addArgument(['--dry'], {
+  action: 'storeTrue',
+  help: 'Dry run. Stage all packages in verdaccio but do not publish them to the registry.',
+});
+
+async function getNewlyCreatedBranches(): Promise<string[]> {
+  const branchesStr = execSync(
+    `git branch -r --sort=-authordate --format='%(HEAD) %(refname:lstrip=-1)'`,
+  ).toString();
+
+  const branches = branchesStr.split('\n').map((line: string) => line.trim());
+
+  const pattern = /^tfjs_\d+\.\d+\.\d+.*$/;
+  const tfjsBranches = branches.filter((branch: string) => pattern.test(branch));
+
+  return tfjsBranches;
+}
+
+async function main() {
+  const args = parser.parseArgs();
+
+  try {
+    const branches = await getNewlyCreatedBranches();
+    console.log('Branches:', branches);
+
+    const latestBranch = branches[0];
+    console.log('Latest Branch:', latestBranch);
+
+    const answer = await question(chalk.cyan.bold(`Is this the right branch '${latestBranch}' you are looking for? (y/n): `));
+
+    if (answer.toLowerCase() === 'y') {
+      checkoutReleaseBranch(latestBranch, args.git_protocol, TMP_DIR);
+      const targetDir = `${TMP_DIR}/tfjs-converter/python`;
+      shell.cd(targetDir);
+      console.log(chalk.blue.bold('Current directory:', shell.pwd().toString()));
+      const executeCommand = (code, stdout, stderr) => {
+        if (code !== 0) {
+          console.error(`Error executing the command: ${stderr}`);
+          return;
+        }
+        console.log(stdout);
+      };
+      shell.exec('bazel clean', executeCommand);
+      shell.exec('bazel build python3_wheel', executeCommand);
```

Comment:
You might be able to use [the `$` function we have in release-util.ts](https://github.com/tensorflow/tfjs/blob/master/scripts/release-util.ts#L236) instead.